tests/foreman/ui/test_repository.py

```diff
@@ -714,6 +714,34 @@ def test_positive_sync_ansible_collection_gallaxy_repo(session, module_prod):
         assert result['result'] == 'success'
 
 
+@pytest.mark.tier2
+def test_positive_no_errors_on_repo_scan(target_sat, function_org):
+    """Scan repos for RHEL Server Extras, then check the production log
+    for a specific error
+
+    :id: 443bf4af-7f9a-48b8-8f98-fdb170e8ae88
+
+    :expectedresults: The specific error isn't contained in the prod log
+
+    :customerscenario: True
+
+    :BZ: 1994212
+
+    :CaseLevel: Integration
+    """
+    target_sat.upload_manifest(function_org.id)
```

Comment:
```suggestion
```

Reply:
This step is performed in the manifester org fixtures, so you can drop a line by using those fixtures instead of calling `upload_manifest` in the test body.